test/sagemaker_tests/pytorch/__init__.py

```diff
@@ -0,0 +1,182 @@
+from base64 import b64decode
+
+
+def get_sagemaker_session(region):
+    return Session(boto_session=boto3.Session(region_name=region))
+
+
+def get_unique_name_from_tag(image_uri):
+    """
+    Return the unique from the image tag.
+
+    :param image_uri: ECR image URI
+    :return: unique name
+    """
+    return re.sub('[^A-Za-z0-9]+', '', image_uri)
+
+
+def get_account_id_from_image_uri(image_uri):
+    """
+    Find the account ID where the image is located
+
+    :param image_uri: <str> ECR image URI
+    :return: <str> AWS Account ID
+    """
+    return image_uri.split(".")[0]
+
+def get_region_from_image_uri(image_uri):
+    """
+    Find the region where the image is located
+
+    :param image_uri: <str> ECR image URI
+    :return: <str> AWS Region Name
+    """
+    region_pattern = r"(us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d+"
+    region_search = re.search(region_pattern, image_uri)
+    assert region_search, f"{image_uri} must have region that matches {region_pattern}"
+    return region_search.group()
+
+
+def ecr_repo_exists(ecr_client, repo_name, account_id=None):
+    """
+    :param ecr_client: boto3.Client for ECR
+    :param repo_name: str ECR Repository Name
+    :param account_id: str Account ID where repo is expected to exist
+    :return: bool True if repo exists, False if not
+    """
+    query = {"repositoryNames": [repo_name]}
+    if account_id:
+        query["registryId"] = account_id
+    try:
+        ecr_client.describe_repositories(**query)
+    except ecr_client.exceptions.RepositoryNotFoundException as e:
+        return False
+    return True
+
+def delete_file(file_path):
+    subprocess.check_output(f"rm -rf {file_path}", shell=True, executable="/bin/bash")
+
+def get_ecr_login_boto3(ecr_client, account_id, region):
+    """
+    Get ECR login using boto3
+    """
+    user_name, password = None, None
+    result = ecr_client.get_authorization_token()
+    for auth in result['authorizationData']:
+        auth_token = b64decode(auth['authorizationToken']).decode()
+        user_name, password = auth_token.split(':')
+    return user_name, password
+
+
+def save_credentials_to_file(file_path, password):
+    with open(file_path, "w") as file:
+        file.write(f"{password}")
+
+class ECRRepoDoesNotExist(Exception):
+    pass
+
+def reupload_image_to_test_ecr(source_image_uri, target_image_repo_name, target_region):
+    """
+    Helper function to reupload an image owned by a another/same account to an ECR repo in this account to given region, so that
+    this account can freely run tests without permission issues.
+
+    :param source_image_uri: str Image URI for image to be tested
+    :param target_image_repo_name: str Target image ECR repo name
+    :param target_region: str Region where test is being run
+    :return: str New image URI for re-uploaded image
+    """
+    ECR_PASSWORD_FILE_PATH = os.path.join("/tmp", f"{get_unique_name_from_tag(source_image_uri)}.txt")
```

Comment:
nit: this is styled like a global, can prob just be normal lowercase variable

Reply:
Same here. The function already exist [here](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/utils/__init__.py#L126), moved a folder above. 
I will take this into consideration when i cleanup the [function](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/utils/__init__.py#L126) in subsequent PR.